lib/metrics/normalizer.js

```diff
@@ -112,45 +110,53 @@ MetricNormalizer.prototype.load = function load(json) {
   }
 }
 
+/**
+ *  Helper for loadFromConfig
+ *
+ * @param rule {object} from this.config.rules
+ * @param ctx {object} context from the calling function
+ */
+const processNameRule = function processName(rule, ctx) {
```

Comment:
Is using an assignment to rename a function something that is less cognitively complex?

Reply:
Apparently so, oddly enough.

Reply:
Ah, wait--I see what you mean. I'd thought you were referring to pulling the processing block into a function. The cognitive complexity total is mostly affected by breaks in flow and nesting. Assignment doesn't seem to move the needle, but I can take it out if it's jarring.

Reply:
Thank you, yes, that's better!